index.js

```diff
@@ -56,13 +56,19 @@ function flatten(arrays) {
 
 
 function getSimulatorInfo(extensionDir) {
+  var binaryDir = currentPlatform.simulatorBinary;
   var simulatorRegex = /fxos_(.*)_simulator@mozilla\.org$/;
   var matches = simulatorRegex.exec(extensionDir);
   if (matches && matches[1]) {
     var version = matches[1].replace('_', '.');
+
+    if (process.platform === 'darwin') {
+      binaryDir = binaryDir['version' + matches[1].replace('_', 'x')] || binaryDir.standardPath;
```

Comment:
Lol yea. jshint fails because using an underscore means that the property names aren't camelCased. Sigh. We could always remove that rule from the jshintrc, although it's my favorite one.